.github/workflows/test-e2e-with-modulereleasemeta.yml

```diff
@@ -55,6 +55,7 @@ jobs:
           - misconfigured-kyma-secret
           - rbac-privileges
           - ocm-compatible-module-template
+          - modulereleasemeta-with-old-moduletemplate-name
```

Comment:
here maybe `modulereleasemeta-with-moduletemplate-in-obsolete`, to indicate this moduletemplate is not an normal scenario.